bin/prism

```diff
@@ -220,7 +220,7 @@ module Prism
       source, filepath = read_source(argv)
 
       ripper = Ripper.sexp_raw(source)
-      prism = Prism::RipperCompat.sexp_raw(source)
+      prism = Prism::Translation::Ripper.sexp_raw(source) rescue :parse_error
```

Comment:
Can we specifically rescue `NotImplementedError` (I'm assuming that's what your handling)? I wouldn't want this to hide a different problem.

```suggestion
      prism =
        begin
          Prism::Translation::Ripper.sexp_raw(source)
        rescue NotImplementedError
          :parse_error
        end
```

Reply:
I mostly use it to make sure that if something's not working yet in Translation::Ripper that I still see CRuby's Ripper output. When it says "parse_error" that should only hide another problem if I'm expecting a parse error. But if I'm midway through making a modification and it might get an error, this means that "prism ripper" is still useful. I guess I could add a mode that doesn't actually use Prism's ripper and just prints out CRuby's output without trying Prism?

Reply:
Ahh that's fine. We can keep it for now, I just want this to go away eventually.